pkg/karmadactl/get/get.go

```diff
@@ -244,6 +251,24 @@ func (g *CommandGetOptions) Validate(cmd *cobra.Command) error {
 	if g.OutputWatchEvents && !(g.Watch || g.WatchOnly) {
 		return fmt.Errorf("--output-watch-events option can only be used with --watch or --watch-only")
 	}
+
+	if len(g.Clusters) > 0 {
+		clusters, err := g.karmadaClient.ClusterV1alpha1().Clusters().List(context.TODO(), metav1.ListOptions{})
+		if err != nil {
+			return err
+		}
+		clusterSet := sets.NewString()
+		for _, cluster := range clusters.Items {
+			clusterSet.Insert(cluster.Name)
+		}
+		errs := []error{}
+		for _, cluster := range g.Clusters {
+			if !clusterSet.Has(cluster) {
+				errs = append(errs, fmt.Errorf("cluster %s does not exist", cluster))
```

Comment:
the error message will be like: `[cluster does not exist:0, cluster does not exist:1, cluster does not exist:2]`, which is not a good format, can we make it better?

Reply:
I am a reference https://github.com/karmada-io/karmada/blob/aeac17b77dc98c5dc74435fad9b5c8dc40b65be8/pkg/karmadactl/apply/apply.go#L134

Reply:
I have considered it. I agree with your opinion and will revise apply after that.

Reply:
```shell
# _output/bin/linux/amd64/karmadactl --kubeconfig /etc/karmada/karmada-apiserver.config get pod -C me1,meqe -A                                                                          [0]
error: [ cluster does not exist: me1, cluster does not exist: meqe ]
```